### DISCUSSION TITLE
Give the root tenant users access to all other tenants

### DISCUSSION THREAD

**Discussion by marcmontecalvo** [Finbuckle/Finbuckle.MultiTenant#615] [Q&A]
Is there a way to give the root tenant users access to all other tenants?  I'm using a shared database and EF.

Thanks!

**Comment by AndrewTriesToCode** [comment]
hi, no built in support but doable by using `IgnoreQueryFilter` on your EF Core queries for such users. You could also modify `MultiTenantDbContext` pretty easily to and change the filter logic it sets up.
